lib/normalize_grammar.rei

```diff
@@ -1 +1,29 @@
-let normalize: Ast_grammar.t => Ast_grammar_normalized.t;
+
+module A = Ast_grammar;
+module B = Ast_grammar_normalized;
+
+let normalize_to_simple: A.rule_body => (
```

Comment:
you don't want to put those functions in the interface file (the .rei).
They should not be used from outside the module, only normalize should.
It's good practice to keep the .rei files small.

Reply:
ok